receiver/zookeeperreceiver/metadata.yaml

```diff
@@ -5,25 +5,34 @@ attributes:
     description: State of the Zookeeper server (leader, standalone or follower).
   zk.version:
     description: Zookeeper version of the instance.
+  follower.state:
+    description: State of followers
+    enum:
+      - synced
+      - not_synced
+  packet.state:
```

Comment:
Could we call this `direction`?

Reply:
Missed this when I circled back. Should be `direction`, not `packet.direction`

Reply:
Yup I'll do a quick fix